From 6465c32b86bfc0ea7360802755adb323d61009bb Mon Sep 17 00:00:00 2001
From: Marcin Wojtas <mw@semihalf.com>
Date: Sun, 1 May 2016 03:07:55 +0200
Subject: [PATCH 0408/2241] arm64: defconfig: enable ASYNC_TX_DMA

This commit enables mv_xor offload when using RAID456.

Fix for SYSTEMSW-2458 "XOR not used in RAID5"

Change-Id: I2b64dd07ee7c13aac63b719cb008da4b5d74595d
Signed-off-by: Marcin Wojtas <mw@semihalf.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/29893
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Nadav Haklai <nadavh@marvell.com>
---
 arch/arm64/configs/mvebu_v8_lsp_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/mvebu_v8_lsp_defconfig b/arch/arm64/configs/mvebu_v8_lsp_defconfig
index 287ea69..8bc71a4 100644
--- a/arch/arm64/configs/mvebu_v8_lsp_defconfig
+++ b/arch/arm64/configs/mvebu_v8_lsp_defconfig
@@ -190,6 +190,7 @@ CONFIG_RTC_DRV_EFI=y
 CONFIG_DMADEVICES=y
 CONFIG_MV_XOR=y
 CONFIG_MV_XOR_V2=y
+CONFIG_ASYNC_TX_DMA=y
 CONFIG_DMATEST=y
 CONFIG_VFIO=y
 CONFIG_VFIO_PCI=y
-- 
2.7.4

